src/persist-client/src/internal/metrics.rs

```diff
@@ -228,14 +226,6 @@ impl MetricsVecs {
                 help: "time spent in external service calls",
                 var_labels: ["op"],
             )),
-            external_consensus_cas_mismatch_versions_count: registry.register(metric!(
```

Comment:
tiny bit of a bummer to lose this coverage, but definitely worth it on the balance